packages/react-native-session-replay/android/src/main/kotlin/com/datadog/reactnative/sessionreplay/ReactTextPropertiesResolver.kt

```diff
@@ -0,0 +1,133 @@
+package com.datadog.reactnative.sessionreplay
+
+import android.view.Gravity
+import android.widget.TextView
+import com.datadog.android.sessionreplay.model.MobileSegment
+import com.datadog.reactnative.sessionreplay.extensions.convertToDensityNormalized
+import com.datadog.reactnative.sessionreplay.extensions.densityNormalized
+import com.facebook.react.views.text.ReactTextShadowNode
```

Comment:
Not a big deal, but I think we could make the `reactTextShadowNodeUtils` better if we abstract everything related to RN in that class.

We don't need to get a `ReactTextShadowNode` in itself here, we can get a custom object that uses the `shadowNode` and the `view` to return the desired values. I believe it would make this file a lot more readable as well :) 

But again, this is a nice to have for me, not mandatory.